### DISCUSSION TITLE
NUE Website axellarator.github.io

### DISCUSSION THREAD

**Discussion by Axellarator** [nuejs/nue#34] [Q&A]
Set up a website based on Tiro's simple template.  [A medical web on Github](https://axellarator.github.io/index.html). You get two pages with stuff in English and German. Nothing fancy. Creation and building is pretty easy and understandable. Modified and enhanced the primary.css file a bit and added a button and some links. Now I am waiting for the things to come. 
Cheers

**Comment by tipiirai** [comment]
WOW! You made it first! So glad to see this.

**Comment by diomed** [comment]
what is that ? I dont understand.

**Reply by tipiirai** [reply]
That's a website made with Nue JS by @Axellarator
